packages/auth-construct/src/string_maps.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable spellcheck/spell-checker */
```

Comment:
why no spellcheck for this file? I'd prefer to ignore individual lines rather than the whole file

Reply:
+1.

What are the words that are picked up ?
If they're ~sane, we can add them to dictionary.

Reply:
'birthdate' is apparently misspelled. I can change it to next-line disable.

@sobolk Is there a way to add words to the dictionary? Where is that list?